service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/RespondentsUpdatedEventHandlerEmailTemplateTest.java

```diff
@@ -55,6 +65,10 @@
         .lastName(RESPONDENT_LAST_NAME).build())
         .build();
     private static final long CASE_ID = 1234567890123456L;
+    public static final String CASE_NAME = "FPL case test";
+
+    @MockBean
+    private FeatureToggleService toggleService;
```

Comment:
toggle doesn't appear to be used, it can be defaulted to true as that is the current state of the toggle in prod. I should make a pr to clean this up

Reply:
feature toggle service is not used but required for EmailNotificationHelper. As you suggested, I added @MockBeans at the class level